drivers/bme280/bme280_saul.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2016 Kees Bakker, SODAQ
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_bme280
+ * @{
+ *
+ * @file
+ * @brief       SAUL adoption for BME280 sensor.
+ *
+ * @author      Kees Bakker <kees@sodaq.com>
+ *
+ * @}
+ */
+
+#include "saul.h"
+
+#include "bme280.h"
+
+static int read_temperature(void *dev, phydat_t *res)
+{
+    bme280_t *d = (bme280_t *)dev;
+
+    res->val[0] = bme280_read_temperature(d);
+    res->unit = UNIT_TEMP_C;
+    res->scale = -2;    /* TODO. Is this correct? The BME function gives us centi Celcius. */
+
+    return 1;
+}
+
+static int read_relative_humidity(void *dev, phydat_t *res)
+{
+    bme280_t *d = (bme280_t *)dev;
+
+    res->val[0] = bme280_read_humidity(d);
+    res->unit = UNIT_PERCENT;
+    res->scale = -2;    /* TODO. Is this correct? The BME function gives us centi percentage. */
+
+    return 1;
+}
+
+static int read_pressure(void *dev, phydat_t *res)
+{
+    bme280_t *d = (bme280_t *)dev;
+
+    res->val[0] = bme280_read_pressure(d) / 100;
```

Comment:
But the "normal" unit for pressure is hPa. If we change the divider and the scale it becomes deca Pa.
Our current phydat_str does not have "h" for scale 2. But I wish we had.